universitat-bremen-institut-fur-politikwissenschaft.csl

```diff
@@ -67,24 +67,44 @@
   </macro>
   <macro name="access">
     <choose>
-      <if type="webpage article-journal article-magazine article-newspaper article speech" variable="URL" match="all">
-        <group>
-          <text value="Text abrufbar unter:" prefix=" " suffix=" "/>
-          <text variable="URL"/>
-          <group prefix=" (" suffix=")">
-            <text value="Zugriff am" suffix=" "/>
-            <date variable="accessed">
-              <date-part name="day" form="numeric" suffix="."/>
-              <date-part name="month" form="numeric" suffix="."/>
-              <date-part name="year" form="long"/>
-            </date>
-          </group>
-        </group>
+      <if variable="URL">
+        <choose>
+          <if type="article-journal" variable="URL DOI" match="all">
+            <choose>
+              <if variable="volume" match="none">
+                <group>
+                  <text value="Text abrufbar unter:" prefix=" " suffix=" "/>
+                  <text variable="URL"/>
+                </group>
+              </if>
+            </choose>
+          </if>
+          <else-if type="article-journal" variable="DOI" match="none">
```

Comment:
The instructions say that if an article has a volume and issue number then the URL should be excluded even if we got it from an internet source (p. 30). In fact, we want to print DOI numbers only for journal articles that are exclusively published online.